companion/src/modeledit/curves.cpp

```diff
@@ -588,6 +565,64 @@ void Curves::on_curveApply_clicked()
   emit modified();
 }
 
+void Curves::ShowContextMenu(const QPoint& pos) // this is a slot
+{
+  QPushButton *button = (QPushButton *)sender();
+  int index = button->property("index").toInt();
+  
+  const QClipboard *clipboard = QApplication::clipboard();
+  const QMimeData *mimeData = clipboard->mimeData();
+  QWidget * mywidget=(QWidget *)sender();
+  QPoint globalPos = mywidget->mapToGlobal(pos);
+
+  QMenu myMenu;
+  QAction *action;
+  action=myMenu.addAction(CompanionIcon("copy.png"),tr("Copy"));
```

Comment:
I would prefer to have more elaborate menus with shorcuts enabled, like it is done already in mixes and logical switches:
```c
    contextMenu.addAction(CompanionIcon("copy.png"), tr("&Copy"),this,SLOT(cswCopy()),tr("Ctrl+C"));
    contextMenu.addAction(CompanionIcon("cut.png"), tr("&Cut"),this,SLOT(cswCut()),tr("Ctrl+X"));
    contextMenu.addAction(CompanionIcon("paste.png"), tr("&Paste"),this,SLOT(cswPaste()),tr("Ctrl+V"))->setEnabled(hasData);
    contextMenu.addAction(CompanionIcon("clear.png"), tr("&Delete"),this,SLOT(cswDelete()),tr("Delete"));
```